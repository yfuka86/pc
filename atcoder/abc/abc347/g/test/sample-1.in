5
0 2 1 0 4
4 0 0 0 2
3 1 0 3 0
1 0 0 0 0
0 0 2 0 5
