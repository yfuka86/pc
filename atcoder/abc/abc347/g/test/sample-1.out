3 2 1 2 4
4 2 2 2 2
3 1 2 3 3
1 1 2 3 4
1 1 2 3 5
